wavelength-plugins/profile.js

```diff
@@ -244,24 +249,24 @@ exports.commands = {
 			}
 			if (fc.length < 12) return this.errorReply("Your friend code needs to be 12 digits long.");
 			fc = fc.slice(0, 4) + '-' + fc.slice(4, 8) + '-' + fc.slice(8, 12);
-			Db.friendcodes.set(toId(user), fc);
-			return this.sendReply("Your friend code: " + fc + " has been saved to the server.");
+			Db.friendcode.set(toId(user), fc);
+			return this.sendReply(`Your friend code: ${fc} has been saved to the server.`);
 		},
 
 		remove: 'delete',
 		delete: function (target, room, user) {
 			if (room.battle) return this.errorReply("Please use this command outside of battle rooms.");
 			if (!user.autoconfirmed) return this.errorReply("You must be autoconfirmed to use this command.");
 			if (!target) {
-				if (!Db.friendcodes.has(toId(user))) return this.errorReply("Your friend code isn't set.");
-				Db.friendcodes.remove(toId(user));
+				if (!Db.friendcode.has(toId(user))) return this.errorReply("Your friend code isn't set.");
+				Db.friendcode.remove(toId(user));
```

Comment:
Another db file name change? Same issues can occur as I mentioned above, it should probably be left as-is.